lib/cli/lib/initiate.js

```diff
@@ -176,17 +157,83 @@ export default function(options, pkg) {
           "Please make sure you are running the `storybook init` command in your project's root directory."
         );
         paddedLog(
-          'You can also install storybook for plain HTML snippets with `storybook init --html` or follow some of the slow start guides: https://storybook.js.org/basics/slow-start-guide/'
+          'You can also install storybook for plain HTML snippets with `storybook init --type html` or follow some of the slow start guides: https://storybook.js.org/basics/slow-start-guide/'
```

Comment:
I think those lines should be changed to something like "you can specify the project type explixitly via ..."